packages/devtools_app/test/test_infra/flutter_test_driver.dart

```diff
@@ -114,53 +112,53 @@ abstract class FlutterTestDriver {
       return -1;
     }
     debugPrint('Sending SIGTERM to $procPid..');
-    Process.killPid(procPid);
-    return proc.exitCode.timeout(quitTimeout, onTimeout: _killForcefully);
+    Process.killPid(procPid!);
+    return proc!.exitCode.timeout(quitTimeout, onTimeout: _killForcefully);
   }
 
   Future<int> _killForcefully() {
     debugPrint('Sending SIGKILL to $procPid..');
-    Process.killPid(procPid, ProcessSignal.sigkill);
-    return proc.exitCode;
+    Process.killPid(procPid!, ProcessSignal.sigkill);
+    return proc!.exitCode;
   }
 
-  String flutterIsolateId;
+  String? flutterIsolateId;
 
-  Future<String> getFlutterIsolateId() async {
+  Future<String?> getFlutterIsolateId() async {
```

Comment:
`vm.isolates` will never be null in practice, but it could be empty so you'll need a check to make sure there's an element before calling `.first`. Also, you can always assume an ID is provided (it's only nullable since tests with mocks like to omit them).

Reply:
as 'first' worked before and it is not production code, skipped validation